### PR TITLE
Create Reset CAN command.

### PR DESCRIPTION
**Pull request by zx100x100**
this command is designed to allow us to request a board resets its self, initial use case is for remote arming, but support should ideally be implemented on every board.

---
This change is [Reviewable](https://reviewable.io/reviews/waterloo-rocketry/canlib/31)

### CODE REVIEW

**Review by jacobdeery — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/canlib/31)* status: 0 of 6 files reviewed, 2 unresolved discussions (waiting on @dwkletke, @Rio6, @wendi-yu, and @zx100x100)

___
*[can_common.c, line 4 at r1](https://reviewable.io/reviews/waterloo-rocketry/canlib/31#-MU0fY1G4xGlzpW8Ge40:-MU0fY1H1138GJuULwpV:b-5g2qzm) ([raw file](https://github.com/waterloo-rocketry/canlib/blob/163a444c45ac84ee2a438bcfe6cce5ee5e41a8e4/can_common.c#L4)):*
> ```C
> #include "message_types.h"
> #include <stddef.h>
> #include <stdio.h>
> ```

Header is unused
___
*[tests/unit/unit_tests.exe, line 0 at r1](https://reviewable.io/reviews/waterloo-rocketry/canlib/31#-MU0fsQu3pmjlmR1Zmyo:-MU0fsQu3pmjlmR1Zmyp:b-hnbr3m) ([raw file](https://github.com/waterloo-rocketry/canlib/blob/163a444c45ac84ee2a438bcfe6cce5ee5e41a8e4/tests/unit/unit_tests.exe#L0)):*
This file should not be tracked; please add `*.exe` to the gitignore

File: can_common.h [hunk 1]
```diff
@@ -169,6 +176,12 @@ bool build_gps_info_msg(uint32_t timestamp,
  */
 int get_general_cmd_type(const can_msg_t *msg);
 
+/*
+*
```
Comment (Rio6) [inline can_common.h:180]: `*`s not aligning is bugging me :p Why is there an empty comment anyways.

**Comment by zx100x100** [comment]
fixed

**Review by Rio6 — APPROVED** [review]
LGTM

File: can_common.h (hunk 1 above)
Comment (Rio6) [reply can_common.h]: You didn't have to change all the comments, it was just me complaining. I was meant to comment on the comment being empty.

There's still some that's not aligned tho, just saying.

**Review by Rio6 — APPROVED** [review]

**Pull request merged**